### PR TITLE
Added Avatar Hairstyles

### PR DESCRIPTION
**Pull request by vikrant1697**
##### Description

I have added 3 girly hairstyles and 2 men hairstyles. 
1. girl_hairstyle1
Screenshot from 2019-03-11 18-09-13
Screenshot from 2019-03-11 18-09-56

---

2. girl_hairstyle2
Screenshot from 2019-03-11 18-10-59
Screenshot from 2019-03-11 18-11-22

---

3. girl_hairstyle3
Screenshot from 2019-03-11 18-16-49
Screenshot from 2019-03-11 18-17-06

---

4. men_hairstyle1
Screenshot from 2019-03-11 18-12-25
Screenshot from 2019-03-11 18-12-43

---

5. men_hairstyle2
Screenshot from 2019-03-11 18-15-44
Screenshot from 2019-03-11 18-16-02

---

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

###### Contributor

- [x] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Tested all changes **locally**
- [x] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/assets/v2/images/avatar/HairStyle/men_hairstyle2-B28E28.svg [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <g
+       id="g862"
+       style="fill:#b28e28;fill-opacity:1;stroke:#333333;stroke-width:15.21328657;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
+    <g
+       id="g864"
+       style="fill:#b28e28;fill-opacity:1;stroke:#333333;stroke-width:15.21328657;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
+    <g
+       id="g866"
+       style="fill:#b28e28;fill-opacity:1;stroke:#333333;stroke-width:15.21328657;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
+  </g>
```
Comment (thelostone-mc) [inline app/assets/v2/images/avatar/HairStyle/men_hairstyle2-B28E28.svg]: Could you compress this ?

Comment (vikrant1697) [reply app/assets/v2/images/avatar/HairStyle/men_hairstyle2-B28E28.svg]: yes done. please review it once :)

**Comment by vikrant1697** [comment]
@thelostone-mc i have compressed all the files. can you take a look :)

**Comment by thelostone-mc** [comment]
@vikrant1697 Ah looks like the rebase went messy ! I see duplicate commits here. Could you ensure your PR has just your commits ?

**Comment by vikrant1697** [comment]
@thelostone-mc hey, since this pr got messy i made another pull request [#4005](https://github.com/gitcoinco/web/pull/4005). please take a look.

**Comment by thelostone-mc** [comment]
closing this out as per above comment

**Pull request closed without merging**